ProjectLighthouse/Pages/UserPage.cshtml

```diff
@@ -120,14 +120,14 @@
 
         @if (!Model.ProfileUser.Banned)
         {
-            <a class="ui red button" href="/admin/user/@Model.ProfileUser.UserId/ban">
-                <i class="ban icon"></i>
-                <span>Ban User</span>
-            </a>
-        
-        <br><br>
+            <div>
+                <a class="ui red button" href="/admin/user/@Model.ProfileUser.UserId/ban">
+                    <i class="ban icon"></i>
+                    <span>Ban User</span>
+                </a>
+            </div>
+            <div class="ui fitted hidden divider"></div>
```

Comment:
What does this look like?

Reply:
![image](https://user-images.githubusercontent.com/16675503/162077732-cbe26a49-2432-4350-aaa0-d0235ac3c5a6.png)